core/src/tangram.cpp

```diff
@@ -257,6 +257,11 @@ void Map::resize(int _newWidth, int _newHeight) {
 
 bool Map::update(float _dt) {
 
+    // Wait until font resources are fully loaded
+    if (impl->scene->fontContext()->resourceLoad > 0) {
```

Comment:
We can do something similar for any uri resource from the scene.
"Wait until texture resources are loaded".